src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/_params.py

```diff
@@ -57,6 +57,8 @@
                       help='The data to use for calling the web service.', required=False)
 register_cli_argument('ml service create realtime', 'num_replicas', options_list=('-z',),
                       default=1, required=False, help='Number of replicas for a Kubernetes service.')
+register_cli_argument('ml service scale realtime', 'num_replicas', options_list=('-z',),
```

Comment:
nit: reduce this to "ml service" level and only have one register_cli_argument. This argument will only apply to functions that have a "num_replicas" arguments, so it's okay to go with broader scope.